wordpress-mobile/WordPress-iOS
# Refactor editor settings to separate class similar to media settings.

To test:
 - Go to App settings and play around with visual editor and native editor switch
 - Native editor option should only show if visual editor is turned on
 - Native editor option should only show in a debug build
 - Start a new post and page and check if editor settings are respected.

Needs review: @diegoreymendez
